test/plugins/globalVariable/index.test.ts

```diff
@@ -0,0 +1,130 @@
+import {GlobalVariablePlugin, factory} from '../../../src/plugins/globalVariable';
+import {Plug} from '../../../src';
+import {PluginSdk} from '../../../src/plugin';
+
+describe('GlobalVariable', () => {
+    let mockPlug: jest.Mocked<Plug>;
+
+    beforeEach(() => {
+        mockPlug = {} as jest.Mocked<Plug>;
+
+        delete window.croct;
+        delete window.onCroctLoad;
+    });
+
+    afterEach(() => {
+        delete window.croct;
+        delete window.onCroctLoad;
+    });
+
+    function createPlugin(): GlobalVariablePlugin {
+        return factory({
+            sdk: {plug: mockPlug} as unknown as PluginSdk,
+            options: {},
+        });
+    }
+
+    it('should set window.croct to the plug instance', () => {
+        const plugin = createPlugin();
+
+        plugin.enable();
+
+        expect(window.croct).toBe(mockPlug);
+    });
+
+    it('should not call any listeners when onCroctLoad is undefined', () => {
+        const plugin = createPlugin();
+
+        plugin.enable();
+
+        expect(window.croct).toBe(mockPlug);
+    });
+
+    it('should call the callback passing the plug', () => {
```

Comment:
```suggestion
    it('should call the listener passing the plug', () => {
```
Once all the other tests use `listener`